src/frontend/citizen-portal/nginx.conf

```diff
@@ -6,13 +6,21 @@ location / {
   add_header Strict-Transport-Security "max-age=31536000; includeSubdomains; preload" always;
   add_header X-Content-Type-Options "nosniff" always;
   add_header Referrer-Policy "no-referrer-when-downgrade";
+
+  add_header Content-Security-Policy "default-src 'self' https://spt.apps.gov.bc.ca; script-src 'self' https://www2.gov.bc.ca 'unsafe-inline' 'unsafe-eval'; font-src 'self' https://*; style-src 'self' 'unsafe-inline';";
```

Comment:
Can we please add a comment here on why each of these are here policies are here.

Reply:
sure